tests/unit/spec/newsletter/newsletter.js

```diff
@@ -0,0 +1,351 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import NewsletterForm from '../../../../media/js/newsletter/newsletter.es6';
+
+describe('NewsletterForm', function () {
+    beforeEach(async function () {
+        const form = `<aside class="mzp-c-newsletter">
+            <div class="newsletter-content">
+                <form id="newsletter-form" class="mzp-c-newsletter-form" action="https://basket.mozilla.org/news/subscribe/" method="post" novalidate>
```

Comment:
The JS tests use this attribute so we can test the error responses. Without it, browsers will use their built-in form validation.

Reply:
ah, I see, ok, browser validation will catch most of the errors that would otherwise show in the error list so that isn't so much an issue here either then

Reply:
Right, the client side error handling is only really there for older browsers, or for cases where something passes native validation but errors in basket.